www/templates/html/Peoplefinder/Developers.tpl.php

```diff
@@ -1,174 +1,117 @@
-        <ul>
+            <h2 id="instance-get"><a href="#instance-get">HTTP GET</a></h2>
+            <p>Returns a representation of the resource, including the properties above.</p>
+            
+            <h2 id="instance-get-example-1"><a href="#instance-get-example-1">Example</a></h2>
+            <ul class="wdn_tabs">
             <?php
-            foreach ($context->resources as $resource) {
-                echo "<li><a href='?view=developers&resource=$resource'>$resource</a></li>";
+            foreach ($resource->formats as $format) {
+                echo "<li><a href='#$format'>$format</a></li>";
             }
             ?>
-        </ul>
+            </ul>
+            <div class="wdn_tabs_content">
+             <?php foreach ($resource->formats as $format): ?>
+                <?php
+                 $url = UNL_Peoplefinder::addURLParams($resource->exampleURI, array('format' => $format));
+                 if (substr($url, 0, 2) == '//') {
+                     $url = 'http:' . $url;
+                 }
+                 ?>
+                 <div id="<?php echo $format; ?>">
+                      <pre><code>GET <?php echo $url; ?></code></pre>
+                      <h3>Response</h3>
+                      <?php
+                      //Get the output.
+                      if (!$result = file_get_contents($url)) {
+                          $result = "Error getting file contents.";
+                      }
+                      switch ($format) {
+                          case "json":
+                              $code = 'javascript';
+                              //Pretty print it
+                              $result = json_decode($result);
+                              $result = json_encode($result, JSON_PRETTY_PRINT);
+                              break;
+                          case "xml":
+                              $code = "xml";
+                              break;
+                          default:
+                              $code = "html";
+                      }
+                      ?>
+                      <pre class="code">
+                          <code class="<?php echo $code; ?>"><?php echo htmlentities($result); ?></code>
+                      </pre>
+                 </div>
+             <?php endforeach; ?>
+            </div>
+        </div>
     </div>
-    <div class="zenbox neutral" style="width:200px">
-        <h3>Format Information</h3>
-        <p>The following is a list of formats used in Directory.</p>
-        <ul>
-            <li><a href='http://www.json.org/'>JSON (JavaScript Object Notation)</a></li>
-            <li><a href='http://en.wikipedia.org/wiki/XML'>XML (Extensible Markup Language)</a></li>
-            <li>Partial - The un-themed main content area of the page.</li>
-        </ul>
+    <div class="bp3-wdn-col-one-fourth">
+        <nav id='resources' aria-label="API Navigation" class="zenbox primary">
```

Comment:
It's odd because this nav then contains a heading element.

In any event, the quote style needs fixed.